projects/npm-tools/packages/npm-scripts/test/utils/parseBnd.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const path = require('path');
+
+const parseBnd = require('../../src/utils/parseBnd');
+
+describe('parseBnd()', () => {
+	function getBndPath() {
+		return path.join(
+			__dirname,
+			'..',
+			'..',
+			'__fixtures__',
+			'utils',
+			'parseBnd',
+			'bnd.bnd'
+		);
+	}
```

Comment:
If you wanted, there is a `getFixture` function that you can use.

[Example usage here](https://github.com/liferay/liferay-frontend-projects/blob/de5eb0773d8c19c86e1bcc4ff2a97f0eb024e5f6/projects/npm-tools/packages/npm-scripts/test/jsp/extractJS.js#L163).

Reply:
I've created a `getFixturePath` function, because I need the path, not the contents. That's probably why the place where I copied this code from was written that way.

I will change it too to let all the code in the same state.